### ISSUE TITLE
Animations lagging (framedrops) on OnePlus devices

### ISSUE THREAD

**Issue by Richard38907** [z-huang/music#1456]
### Checklist

- [X] I am able to reproduce the bug with the [latest debug version](https://github.com/z-huang/InnerTune/actions/workflows/build.yml).
- [X] I've checked that there is no open or closed issue about this bug.
- [X] This issue contains only one bug.
- [X] The title of this issue accurately describes the bug.

### Steps to reproduce the bug

Try to swipe up the mini player from the playback bar and see the frame drops. This bug is device specific because it is smooth on Pixel 7 but not on OnePlus (Snapdragon) devices. It only happens in pure black mode.

### Expected behavior

Fluid player opening

### Actual behavior

Lagging when pure black mode activated

### Screenshots/Screen recordings

https://github.com/user-attachments/assets/23760f98-6694-488b-9aa4-2980ae36e4ac

The video is 24 fps due to GitHub size restrictions. So you cannot see the lag here.

### Logs

[logcat.txt](https://github.com/user-attachments/files/16739420/logcat.txt)

### InnerTune version

0.5.8

### Android version

Android 13 and 14 (tested)

### Additional information

Tested on:
OnePlus Nord (Avicii)
OnePlus 10t

**Comment by z-huang** [comment]
Since it's device-specific, maybe the problem lies on your device?

**Comment by Richard38907** [comment]
Tested on OnePlus Nord and 10t
Another report from Malopied's fork with OnePlus Nord N300: https://github.com/Malopieds/InnerTune/issues/304

**This problem only happens on OnePlus devices**

**Comment by Richard38907** [comment]
Before the new MD3 update, this glitch didn't happen.
At the moment, all InnerTune versions and forks are impacted by the bug since this new implementation.

**Comment by Richard38907** [comment]
It seems to be fixed for the OnePlus Nord on the latest 0.5.10 version. But the frame drop is still present on my OnePlus 10t.

**Comment by z-huang** [comment]
Well, I didn't do anything to fix this in the latest update...

**Comment by Richard38907** [comment]
Yep. After a clean reinstall of InnerTune, the lag is back on my Nord 🫤.

**Comment by Richard38907** [comment]
Tested on Samsung A40. Same bug since the update.

**Comment by ghost** [comment]
this also happens on my Realme 11 Pro +
